news/2024.html

```diff
@@ -30,6 +30,57 @@
 </div>
 -->
 
+<!----------------------------------------------------------->
+<div class="news-item tag-link" id="summit-2024-keynotes">
+  <a href="#summit-2024-keynotes"><span class="tag-span"># </span></a> 12 June 2024: Keynotes for seL4 summit 2023 announced
+</div>
+<div>
+  <div class="row" style="margin-left:0px; margin-right:0px">
+  <p>
+    We are pleased to announce that the two keynotes for the <a href="../Foundation/Summit/2024">seL4 summit 2024</a> will be <strong>Darren Cofer from Collins Aerospace</strong> and <strong>Ning Qu from
+    NIO</strong>. Darren will talk about <a href="../Foundation/Summit/2024/abstracts2024.html#a-inspecta">Industrial Scale Proof Engineering for Critical Trustworthy Applications (INSPECTA)</a> and Ning about <a href="../Foundation/Summit/2024/abstracts2024.html#a-"><mark>TBC</mark></a>.
+  </p>
+  <p>
+    <strong>Darren Cofer</strong> is ...
+  </p>
+  <p>
+    <strong>Ning Qu</strong> has ...
+  </p>
+  </div>
+  <div class="keynotes_grid">
+    <div class="speaker">
+      <div class="speaker_pic">
+        <img src="../images/summit/darren.jpg" style="width: 150px" alt="Darren Cofer" />
+      </div>
+      <div class="speaker_title" style="text-align:center">
+        <strong>Darren Cofer</strong>
+        <br>
+        Collins Aerospace
```

Comment:
Again (if easy), I would have a link from Darren's picture and name to his page/linkedIn and a link from Collins Aerospace to the company's website.

Same for Ning/NIO